test-results.json

```diff
@@ -1 +1 @@
Users/bakunyo/workspace/github.com/bakunyo/styled-components/src/utils/test/consolidateStreamedStyles.test.js","startTime":1517970999466,"status":"passed","summary":""},{"assertionResults":[{"failureMessages":[],"status":"passed","title":"should let you use another component in a css rule"}],"endTime":1517970999666,"message":"","name":"/Users/bakunyo/workspace/github.com/bakunyo/styled-components/src/test/overriding.test.js","startTime":1517970999571,"status":"passed","summary":""},{"assertionResults":[{"failureMessages":[],"status":"passed","title":"blindly interleave"},{"failureMessages":[],"status":"passed","title":"should be driven off the number of interpolations"}],"endTime":1517970999726,"message":"","name":"/Users/bakunyo/workspace/github.com/bakunyo/styled-components/src/utils/test/interleave.test.js","startTime":1517970999639,"status":"passed","summary":""},{"assertionResults":[{"failureMessages":[],"status":"passed","title":"defaults to reusing the component displayName"},{"failureMessages":[],"status":"passed","title":"falls back to the class name"},{"failureMessages":[],"status":"passed","title":"ultimately falls back to \"Component\""}],"endTime":1517970999821,"message":"","name":"/Users/bakunyo/workspace/github.com/bakunyo/styled-components/src/utils/test/getComponentName.test.js","startTime":1517970999756,"status":"passed","summary":""},{"assertionResults":[{"failureMessages":[],"status":"passed","title":"replaces characters that could be part of CSS selectors"},{"failureMessages":[],"status":"passed","title":"replaces double hyphens with a single hyphen"},{"failureMessages":[],"status":"passed","title":"removes extraneous hyphens at the ends of the string"}],"endTime":1517970999919,"message":"","name":"/Users/bakunyo/workspace/github.com/bakunyo/styled-components/src/utils/test/escape.test.js","startTime":1517970999855,"status":"passed","summary":""},{"assertionResults":[{"failureMessages":[],"status":"passed","title":"should take precedence over ThemeProvider"},{"failureMessages":[],"status":"passed","title":"should fallback to default theme"}],"endTime":1517970999936,"message":"","name":"/Users/bakunyo/workspace/github.com/bakunyo/styled-components/src/utils/test/determineTheme.test.js","startTime":1517970999837,"status":"passed","summary":""},{"assertionResults":[{"failureMessages":[],"status":"passed","title":"should create alphabetic names for number input data"}],"endTime":1517971000068,"message":"","name":"/Users/bakunyo/workspace/github.com/bakunyo/styled-components/src/utils/test/generateAlphabeticName.test.js","startTime":1517970999977,"status":"passed","summary":""},{"assertionResults":[{"failureMessages":[],"status":"passed","title":"should have all valid HTML5 elements defined as properties"}],"endTime":1517971000229,"message":"","name":"/Users/bakunyo/workspace/github.com/bakunyo/styled-components/src/constructors/test/styled.test.js","startTime":1517971000056,"status":"passed","summary":""}],"wasInterrupted":false}
```

Comment:
This won't need to be commited (it's only still in the repo accidentally due to PRs readding it)

But we do need an entry to the changelog